src/app/views/components/metrics.tsx

```diff
@@ -30,9 +29,14 @@ const useStyles = makeStyles((theme) => {
       padding: 2,
     },
     cardHeaderText: {
-      color: indigo[300],
+      color: "#FFF",
```

Comment:
i was wondering if we should change all of the places we have #fff or white to just "white"? but looks great, thank you